### ISSUE TITLE
won't jump to package __init__

### ISSUE THREAD

**Issue by kaldown** [fannheyward/coc-pyright#248]
Having this structure:
```
- A/
-- __init__.py
-- models/
---- __init__.py
```
```
# A/__init__.py
1 from A import models
2 from A.models import db
```

Setting cursor or `models` (1 row)  and hitting `GoToDefinitionOrDeclaration` won't jump. But rather reach top of the current buffer.
Notes:
1) Assuming it should work since no error produced, just wrong cursor set.
2) it works on *w0rp/Ale*
3) it works when I hit `GoToDefinitionOrDeclaration` on an instance from `models.db` (2 row)

**Comment by fannheyward** [comment]
> GoToDefinitionOrDeclaration

Does this is your custom command? coc doesn't provide this.

> on an instance from models.db

The jump location is came from Pyright, looks like Pyright can't find definition. Can you reproduce this with VSCode + https://marketplace.visualstudio.com/items?itemName=ms-pyright.pyright?

**Comment by kaldown** [comment]
> > GoToDefinitionOrDeclaration
> 
> Does this is your custom command? coc doesn't provide this.
> 
> > on an instance from models.db
> 
> The jump location is came from Pyright, looks like Pyright can't find definition. Can you reproduce this with VSCode + https://marketplace.visualstudio.com/items?itemName=ms-pyright.pyright?

Unfortunately I have no experience with `VSCode`.
But I provided the behavior of a similar project `https://github.com/dense-analysis/ale`, which is works as expected.
Just want to give a try to `CoC` ecosystem

**Comment by kaldown** [comment]
`nmap <silent> <Leader>d <Plug>(coc-definition)` to be precise.

**Comment by fannheyward** [comment]
Tested with a similar project:

```
main.py
test/__init__.py
test/m.py
```

```python
#!/usr/bin/env python3

from test import m
```

`coc-definition` on `m` will jump to `test/m.py`

**Comment by kaldown** [comment]
> Tested with a similar project:
> 
> ```
> main.py
> test/__init__.py
> test/m.py
> ```
> 
> ```python
> #!/usr/bin/env python3
> 
> from test import m
> ```
> 
> `coc-definition` on `m` will jump to `test/m.py`

That's correct. But my example is different.
models itself has `__init__.py`
And when I `gd` to `models` I want to look through `__init__.py`

**Comment by fannheyward** [comment]
Properly is a Pyright issue, the definition location is came from Pyright, coc-pyright just do jumping.

You can test with VSCode + https://marketplace.visualstudio.com/items?itemName=ms-pyright.pyright to check whether the jumping is same.

**Comment by kaldown** [comment]
your module is a client. I get it.
Questions like such, should rather being addressed to MS server side.
Thanks.

**Comment by kaldown** [comment]
I don't get then.
If this module is about `vim + coc`. Why should I test `VSCode` then?

https://github.com/microsoft/pyright/issues/1357#issuecomment-757651033
looks like it works in `VSCode`.

Which part isn't working then?

**Comment by yaegassy** [comment]
@fannheyward @kaldown 

It seems that 1.1.101 of pyright has been released. 

https://github.com/microsoft/pyright/releases/tag/1.1.101

It also seems to contain a fix for this issue.

**Comment by fannheyward** [comment]
Try `:CocUninstall coc-pyright` and `:CocInsttall coc-pyright`.
